SalesforceCommerceCloud/pwa-kit
# Remove large space between add to cart and add to wishlist buttons

Remove the unexpected space between add to cart buttons, add shadow to the fixed mobile add to cart section.

[W-9755369](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07AH000000U4gyYAC/view)

deployed here: https://scaffold-pwa-test.mobify-storefront.com/

## How to test-drive this PR
- cd packages/pwa && npm start
- go to `http://localhost:3000/en/product/25588993M`
- verify the add to cart buttons look good

## Changes
- remove margins
- add shadow

## Todos

### General

### Backwards Compatibility

> Breaking changes include:
> * Removing a public function/Component/Prop
> * Adding required arguments to functions
> * Changing the data-types of function params or return values

### Accessibility Compliance
You must check off all items in **one** of the follow two lists:

_or..._


### Documentation


> Documentation and comments should follow the [Mobify Content Style Guide](https://docs.google.com/a/mobify.me/document/d/1jlcg5boC3MUHN7fy2n3Yu_FBGzGgTUPrnMoePzQtqE4/edit?usp=sharing) and [Mobify's Capitalization and Spelling List](https://docs.google.com/document/d/1LO7RAr2vD3LFs_bj5j0vIFMKsVFrOFfEjpCheYEH3Lg/edit?usp=sharing)
